redhat-developer/developers.redhat.com
# [RHDX-309] Add Kiosk Mode option

### JIRA Issue Link
[RHDX-309](https://projects.engineering.redhat.com/browse/RHDX-309)

### Verification Process
1. Sign in and edit the page /eventtutorials/
2. See the Kiosk Mode field option. Check it to enable and publish the change
3. View the page and verify it has a minimal header
4. Remove the setting and publish; Verify the page updates to show the full website header
5. View other content types and non-nodes (assemblies, term, user pages) and verify there are no new or introduced php warnings
